components/search/TierToggler.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div>
+    <p class="font-italic">
+      {{ toggleText }}
+      <nuxt-link
+        :to="togglePath"
+        class="toggle-button"
+        data-qa="tier toggle button"
+        @click.native="toggleHandler"
+      >
+        {{ !active ? button.show : button.hide }}.
+      </nuxt-link>
+    </p>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'TierToggler',
+
```

Comment:
This is something we might want to discuss as a team as well when talking about consistency: having an extra line between options or not.

Reply:
I prefer the extra line as I believe it makes the code look cleaner and easier to read. But I am happy to go with the majority.